src/dvc_render/vega_templates.py

```diff
@@ -205,6 +205,12 @@ class ConfusionTemplate(Template):
         "data": {"values": Template.anchor("data")},
         "title": Template.anchor("title"),
         "facet": {"field": "rev", "type": "nominal"},
+        "params": [
+            {
+                "name": "toggleText",
```

Comment:
[Q] Could we change this to be a single word like "values" to match "smooth" from the smooth/linear template?

Reply:
or `showValues`... unsure, just suggestions

Reply:
Yeah, I'll go with `showValues`. I don't like it either but couldn't find a way to have a nicely formatted name here.